MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+"""
+PNG decoding and encoding library
+"""
+
+module(
+    name = "libspng",
+    version = "0.7.4",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "zlib", version = "1.3.1")
```

Comment:
I'm not familiar with bazel, does this specify 1.3.1 and only 1.3.1 or the minimum version?

Reply:
If `libspng` is the only user of `zlib` in a bazel workspace, then it would mean 1.3.1 exactly.
If there is another `bazel_dep` with, say, `zlib` 1.3.2 in its dependencies, then
- if 1.3.1 and 1.3.2 have the same `compatibility_level` in their definition (here: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/zlib/1.3.1/MODULE.bazel#L4 ), the later 1.3.2 version is pulled;
- otherwise, the dependency graph cannot be resolved.

Full details:
- https://bazel.build/external/module#version-selection
- https://bazel.build/external/module#compatibility_level